gui/default/syncthing/core/syncthingController.js

```diff
@@ -1532,6 +1532,12 @@ angular.module('syncthing.core')
             delete $scope.folderRejections[folder + "-" + device];
         };
 
+        $scope.ignoreRejectedFolder = function (folder, device) {
+            $scope.config.ignoredFolders.push(folder.id);
```

Comment:
On why the button isn't working Could it be that I'm using folder.id here?  I do want only a string and not the whole folder struct in the portion of the config I wrote.  But maybe I'm not understanding how this feature works.